### DISCUSSION TITLE
Port forwarding to Minikube ingress without needing kubectl port-forward

### DISCUSSION THREAD

**Discussion by danijam** [devcontainers/community#85] [Q&A]
FYI I posted this as an issue here: https://github.com/devcontainers/features/issues/756 probably should have posted to this discussion area first, but found it after posting! Appreciate any help.

In summary I successfully created a dev container that comes up with Minikube (See my issue for devcontainer.json). I can successfully `minikube start` and I can successfully `kubectl apply` a hello world web app as a deployment with a service and ingress.

The ingress comes up using the `minikube ip` and I can access the app successfully from my dev container. But I can't figure out how to access from Windows using Visual Studio Port forward.

What I'm trying todo here is create an experience when you can clone my repo, build and drop into the dev container which will spin up the cluster and apply some workloads to it and allow me to bring up my browser on my windows desktop and point to the workloads in the cluster that is running in the dev container.

Trying to avoid having to script lots of kubectl port-forward executions. I'd like the ingress to just be directly on the dev container as a port. Is this possible?

**Comment by samruddhikhandale** [comment]
Responded in https://github.com/devcontainers/features/issues/756#issuecomment-1815536728 Thanks!
